web/styles/popovers.css

```diff
@@ -1015,18 +1015,13 @@ ul {
 
         .dropdown-widget-button {
             /* Override the default border-radius to properly align
```

Comment:
Please remove this entire comment, it's wrong now (we do not override border radius anymore)